Kitodo/src/main/webapp/WEB-INF/templates/includes/tasks/taskList.xhtml

```diff
@@ -65,21 +65,25 @@
         <p:column headerText="#{msgs.actions}">
             <h:form id="actions">
                 <!-- assign task -->
-                <h:commandLink id="take" action="#{CurrentTaskForm.takeOverTask}"
+                <h:commandLink id="take"
+                               actionListener="#{CommentForm.setProcessId(item.process.id)}"
```

Comment:
Why do you need to set the processId on the commentform on every action link?

Reply:
Because I have to set in the 'CommentForm' what is the selected process from the dataTable.